genome_browser/_genome_diagram.py

```diff
@@ -88,25 +90,24 @@ def draw(self):
 
                 # Figure annotations will be applied to the last ax in offset
                 # coordinates in a lightgray text. Clipping is ignored as the
-                # text clearly cips with the axes outboard frame.
-                if self.annotation is not None:
```

Comment:
The `self.annotation = ""` default grants us the same functionality with :one: less code && :two: less worry.